### PR TITLE
refactor(global_plugins): kong to apisix global rules module

### PR DESCRIPTION
**Pull request by shuaijinchao**

### CODE REVIEW

File: pkg/kong/globalrules_test.go [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+	assert.Equal(t, kongPluginKeyAuth.ID, apisixConfig.GlobalRules[0].ID)
+
+	kongConfig = Config{}
+	apisixConfig = apisix.Config{}
+	var kongPluginRateLimitingSecond Plugin
+	kongPluginRateLimitingSecond.ID = uuid.NewV4().String()
+	kongPluginRateLimitingSecond.Name = PluginRateLimiting
+	kongPluginRateLimitingSecond.Enabled = true
+	kongPluginRateLimitingSecondConfig := make(map[string]interface{})
+	kongPluginRateLimitingSecondConfig["second"] = 5
```
Comment (spacewander) [inline pkg/kong/globalrules_test.go:40]: Better to use table drive test: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests

Comment (shuaijinchao) [reply pkg/kong/globalrules_test.go]: done

File: pkg/kong/globalrules_test.go [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+			Name:    PluginRateLimiting,
+			Enabled: true,
+			Config: map[string]interface{}{
+				"year":   5,
+				"policy": PluginRateLimitingPolicyLocal,
+			},
+		},
+	}
+
+	for _, kongPluginRateLimiting := range kongPluginRateLimitings {
```
Comment (spacewander) [inline pkg/kong/globalrules_test.go:100]: We can improve it by providing an "expect" struct, which contains time units like "second" and expected value.

Comment (shuaijinchao) [reply pkg/kong/globalrules_test.go]: OK, done

**Pull request merged**